### ISSUE TITLE
TypeError in _build_bot function with pydicom.filebase.DicomBytesIO

### ISSUE THREAD

**Issue by denesd** [ImagingDataCommons/highdicom#313]
In pydicom 3.0.1 `DicomBytesIO`'s and `DicomFileLike` `read` function changed an only accepts one argument. The current implementation https://github.com/ImagingDataCommons/highdicom/blob/master/src/highdicom/io.py#L188 seems to be compatible with the 2.4.4 based `DicomIO`.

The following error is raised:
```
first_two_bytes = fp.read(2, True)
E           TypeError: read expected at most 1 argument, got 2
```

**Comment by CPBridge** [comment]
Thanks for reporting this issue @denesd, hopefully this should be a simple fix. I'll take a look in the next few days

**Comment by CPBridge** [comment]
Hi @denesd, could you please test whether #314 fixes your issue?

**Comment by denesd** [comment]
Hi @CPBridge , I've tested the fix and it seems to be working as expected :100:

**Comment by CPBridge** [comment]
Closed by #314
